Orange/widgets/classify/owknn.py

```diff
@@ -38,6 +39,13 @@ def __init__(self):
                      items=["Euclidean", "Manhattan", "Maximal", "Mahalanobis"])
         self.metrics = ["euclidean", "manhattan", "chebyshev", "mahalanobis"]
 
+        box = gui.widgetBox(box, "Weight")
+        box.setFlat(True)
+
+        gui.comboBox(box, self, "weight_index",
+                     items=["Uniform", "Distance"])
+        self.weights = ["uniform", "distance"]
```

Comment:
Please specify this list of options once, and reuse it. E.g. including the box, you can do something like:
```py
gui.comboBox(self.controlArea, self, 'weight_type',
             box='Neighbors weight',
             items=[i.capitalize() for i in self.weights])
```